simple-social-icons.php

```diff
@@ -520,9 +520,21 @@ function get_icon_markup( $icon, $label ) {
 		return $markup;
 	}
 
+    /**
+     * Remove option when uninstalling the plugin.
+     *
+     * @since 2.0.2
```

Comment:
Should this be `@since 2.1.0`?

Reply:
Yeap, that's correct! Resolved in the new PR.